docs/source/management/index.md

```diff
@@ -108,16 +112,104 @@ Available SRE configurations for context 'green':
 ```
 
+::::{admonition} Tearing down an SRE deletes ALL data and resources
+Running `dsh sre teardown` will remove all resources deployed within the SRE.
+Ensure that any desired outputs have been egressed before deleting the SRE.
+Any data remaining on the SRE will be deleted.
+The user groups for the SRE on Microsoft Entra ID will also be deleted.
+::::
+
 - Run the following if you want to teardown the deployed SHM:
 
 ```{code} shell
 $ dsh shm teardown
 ```
+
+::::{admonition} Tearing down an SHM
+All SREs associated with the SHM must be torn down before the SHM can be torn down.
+Tearing down the SHM deletes any remotely stored SRE configurations.
+::::
+
+## Managing data ingress and egress
+
+### Data Ingress
+
+It is the data provider's responsibility to upload the data required by the safe haven.
+
+```{important}
+Any data ingress must be signed off by the {ref}`role_data_provider_representative`, {ref}`role_investigator` and referee (if applicable).
+```
+
+The following steps show how to generate a temporary write-only upload token that can be securely sent to the data provider, enabling them to upload the data:
+
+- In the Azure portal select `Subscriptions` then navigate to the subscription containing the relevant SHM
+- Search for the resource group: `shm-<SHM ID>-sre-<SRE ID>-rg`, then click through to the storage account called: `sh<first three letter of SHM ID>sre<first three letters of SRE ID>sensitivedata`
+- Click `Networking` under `Settings` and ensure that the data provider's IP address is one of those allowed under the `Firewall` header, then hit the save icon in the top left
+- From the `Overview` tab, click the link to `Data storage` and then `Containers` (in the middle of the page)
+- Click `ingress`
+- Click `Shared access tokens` under `Settings` and do the following:
+    - Under `Signing method`, select `User delegation key`
+    - Under `Permissions`, check these boxes:
+        - `Write`
+        - `List`
+    - Set a 24 hour time window in the `Start and expiry date/time` (or an appropriate length of time)
+    - Leave everything else as default and click `Generate SAS token and URL`
+    - Copy the `Blob SAS URL`
+
+      ```{image} ingress_token_write_only.png
+      :alt: write-only SAS token
+      :align: center
+      ```
+
+- Send the `Blob SAS URL` to the data provider through a secure channel (for example, you could use the [Egress secure email](https://www.egress.com/) service)
+- The data provider should now be able to upload data
+- You can validate successful data ingress by logging into a workspace in the SRE and checking the `/mnt/input` volume, where you should be able to view the data that the data provider has uploaded
+
+### Data egress
+
+```{important}
+Any data egress must be signed off by the {ref}`role_data_provider_representative`, {ref}`role_investigator` and referee (if applicable).
+```
+
+```{important}
+Assessment of output must be completed **before** an egress link is created.
```

Comment:
I think we probably can say this. It might help to say why though.

Reply:
Will just remove if we're not talking about classification etc

Reply:
I think it would be good to keep. I can't imagine a functional TRE where there isn't some sort of output checking.

It's just that in these docs we need to talk about how DSH works and how to use it, not enforce a particular IG process. So I think the important point here to to highlight that outputs may be sensitive so you must take care, consider the outputs and have _some_ process.

I think talking about classification isn't possible because we either leave it undefined ("You must do classification", but what does classification mean?) or we define a classification process, which is definitely out of scope.